about.html

```diff
@@ -0,0 +1,114 @@
+			<li class="navbar"><a class="nav-link" href="./index.html">Home</a></li>
+			<li class="navbar"><a class="nav-link" href="#section_2_about">Events</a></li>
+			<li class="navbar"><a class="nav-link" href="#section_3_about">Tours</a></li>
+			<li class="navbar"><a class="nav-link" href="footer">Contact</a></li>
+		</ul>
+	</header>
+	<section class="section_1_about" id="section_1_about">
+		<h2 class="main-text-about">
+			Your Perfect Tour in Uzbekistan Starts Here!
+		</h2>
+		<p class="text-about">
+			Lorem Ipsum Travel has passed a long way since its early beginnings in 1988. The company was founded by
+			two ordinary guys passionate about tourism. At the moment, the company serves more than 15,000 travellers
+			across Central Asia each year.
+
+			Our professional team has been lucky to travel to a number of amazing countries across all continents and
+			can proudly boast many years of practical expertise in the travel industry. They are committed to
+			excellence in providing clients with high-quality travel and the best possible traveling experience at
+			reasonable prices. Due to the time spent abroad, they clearly understand all the demands of a modern
+			traveler. Our specialists are always friendly and pay individual attention to each client. Expert guides
+			will happily share with you all of their favourite spots
+		</p>
+	</section>
+
+	<section class="section_2_about" id="section_2_about">
+		<h2 class="text-2-about">Our vision</h2>
+		<div class="line-separator3"></div>
+		<p class="text-2-text">Lorem Ipsum has come a long way since its humble beginnings in 1997. A company
+			that began with two bearded backpackers, a typewriter and a kitchen table now leads 10,000 travelers
+			across
+			the Europe each year. Our team have been lucky enough to travel to a huge variety of incredible countries
+			across all seven continents, and collectively boast many years of experience in the travel industry to
+			compliment our modern approach to providing quality travel.
+			Because of our time spent abroad, we know exactly what the modern traveller wants from a destination
+			specialist. Our specialists offer an honest, informed approach to planning your holiday and we return
+			regularly to our offered regions to check on any new products and destinations and to conduct a quality
+			check
+			of the products we already know and love.</p>
+	</section>
+
+	<section class="section_3_about" id="section_3_about">
+		<h2 class="text-3-about">See the last Tours</h2>
+		<div class="line-separator3"></div>
+		<div class="images-div">
+			<div class="image-div">
+				<img class="red-img" src="./assets/images/Red-mountains.png" alt="">
+				<p class="img-text">2018 <br> Best Mountains hiking marathon</p>
+			</div>
+			<div class="image-div">
+				<img class="red-img2" src="./assets/images/Red-samarkand.png" alt="">
+			</div>
+			<p class="img-text2">2019 <br> Sharq Taronalari</p>
+		</div>
+	</section>
+
+
+
```

Comment:
- [ ] OPTIONAL
To enhance a cleaner code kindly ensure not to leave extra lines and spacing which are serving no purpose.